saleor/payment/interface.py

```diff
@@ -82,6 +90,15 @@ class PaymentData:
     reuse_source: bool = False
     data: Optional[dict] = None
     graphql_customer_id: Optional[str] = None
+    # Optional, lazy-evaluated gateway arguments
+    _resolve_lines: InitVar[Callable] = None
+
+    def __post_init__(self, _resolve_lines: Callable):
+        self.__resolve_lines = _resolve_lines
+
+    @cached_property
+    def lines(self) -> List[PaymentLineData]:
```

Comment:
FYI - this is not visible to `dataclasses.asdict` so this field does not appear in webhook payloads. Maybe it would be worth leaving a comment about this.